packages/utils/src/graphql/graphql.spec.ts

```diff
@@ -291,4 +291,18 @@ describe('utils that handle schema.graphql', () => {
     expect(entities.models?.[0].indexes[0].fields).toEqual(['field2']);
     expect(entities.models?.[0].indexes[1].fields).toEqual(['field3']);
   });
+
+  it('can read composite index', () => {
+    const graphqlSchema = gql`
+      type StarterEntity @entity {
+        id: ID! #id is a required field
+        field1: Int!
+        field2: String #field2 is an optional field
+        joinIndex1: CompositeIndex @joinIndex(fields: ["field1", "field2"])
+      }
```

Comment:
It seems weird to have another field. I would have thought it would be on the entity level like this:

```suggestion
      type StarterEntity @entity @index(fields: ["field1","field2"]) {
        id: ID! #id is a required field
        field1: Int!
        field2: String #field2 is an optional field
      }
```

or

```suggestion
      type StarterEntity @entity(index: ["field1","field2"]) {
        id: ID! #id is a required field
        field1: Int!
        field2: String #field2 is an optional field
      }
```

Reply:
In this way, we only able to defined one composite index per entity, which is ["field1","field2"].